warehouse/forklift/legacy.py

```diff
@@ -16,6 +16,7 @@
 import re
 import tempfile
 import zipfile
+from cgi import parse_header
 
 from cgi import FieldStorage
```

Comment:
Seems weird to use old cgi module functions where Pyramid provides the needed parsers.

Reply:
@merwok Thanks for the review. Assuming you're talking about `parse_header`, would you mind pointing me to the equivalent function which Pyramid provides?

Reply:
Sorry for missing this!  I’m talking about both parse_header and FieldStorage; Pyramid parses these and surely exposes a way to get Python objects for them, but I can’t look for it right now.